### PR TITLE
Update landing page

### LINKED ISSUE #2949: Website: Landing page release video positioning - Light Weight Experiments
We need to figure out how we should add the explainer videos for releases to the landing page of the corresponding website for the release for a period of time.

Should we:

1. Just put where the current video is?
2. Put higher up on page in some kind of New! block?
3. Other ideas

We will be trying this out for the first time for the **November 17th** Release of Lightweight Experiments - (which will likely be re-named) So that's the deadline.

### PR DESCRIPTION
**Pull request by julieg18**
* added desktop version of the modal to the tablet and desktop
* adjusted WhatNew modal to fit inside phone 
* added video to hero

Fixes #2949

### CODE REVIEW

File: src/components/Home/WhatsNewModal/index.tsx
Comment (julieg18) [inline src/components/Home/WhatsNewModal/index.tsx:81]: Does the title or paragraph text need revising?

File: src/components/Home/WhatsNewModal/index.tsx
Comment (julieg18) [inline src/components/Home/WhatsNewModal/index.tsx:87]: We need to update this link to the blog post link.

File: src/components/Home/WhatsNewModal/index.tsx
Comment (julieg18) [inline src/components/Home/WhatsNewModal/index.tsx:83]: We'll need to update this Video id to the actual video.

File: src/components/Home/WhatsNewModal/styles.module.css
Comment (julieg18) [inline src/components/Home/WhatsNewModal/styles.module.css:17]: We don't have tablet or mobile versions of the model yet. For now, I kept the model in the tablet screens and took it off for the mobile screens.

File: src/components/Home/WhatsNewModal/styles.module.css
Comment (julieg18) [inline src/components/Home/WhatsNewModal/styles.module.css:50]: I feel like the animation for the blue button could be better, but I'm not sure how to improve it. 🤔

File: src/components/Home/WhatsNewModal/index.tsx
Comment (dberenbaum) [inline src/components/Home/WhatsNewModal/index.tsx:13]: I don't think we need version for now. It's only for this specific "release," which is not strongly tied to new features in any DVC version.

Comment (julieg18) [reply src/components/Home/WhatsNewModal/index.tsx:13]: Makes sense! Would the button saying "What's New" work?

File: src/components/Home/WhatsNewModal/index.tsx
Comment (julieg18) [reply src/components/Home/WhatsNewModal/index.tsx:81]: Do we have a link yet? I can add the link if we do!

File: src/components/Home/WhatsNewModal/index.tsx
Comment (dberenbaum) [reply src/components/Home/WhatsNewModal/index.tsx:13]: Yup, "What's new" works!

File: src/components/Home/WhatsNewModal/styles.module.css
Comment (julieg18) [inline src/components/Home/WhatsNewModal/styles.module.css:129]: I went ahead and adjusted the model and button positions slightly so the modal could atleast fit inside phone screens.

File: src/components/Home/WhatsNewModal/index.tsx
Comment (dberenbaum) [reply src/components/Home/WhatsNewModal/index.tsx:79]: @jendefig pointed out that we might not actually need the video in the modal since it will already be in the hero, and it could be redundant and excessive to put it in both.

Comment (jendefig) [reply src/components/Home/WhatsNewModal/index.tsx:79]: I think directing to Dave's blog post will make the most sense.

File: src/components/Home/WhatsNewModal/styles.module.css
Comment (jendefig) [reply src/components/Home/WhatsNewModal/styles.module.css:46]: I think animation on the button would be cool and draw attention to it.  Also when I looked at the Heroku on desktop.  It feels like the button is a bit too far down on the page.  Might be better up a skosh, into the first section, or even on the line between the first and second section.  That might call attention to it more.
image

Comment (julieg18) [reply src/components/Home/WhatsNewModal/styles.module.css:46]: We could move the button higher up but we'd have to move the button lower back to the corner of the screen when the user starts scrolling. That may end up looking glitchy 🤔

File: src/components/Home/WhatsNewModal/index.tsx
Comment (dberenbaum) [reply src/components/Home/WhatsNewModal/index.tsx]: Okay, we've decided on https://dvc.org/blog/ml-experiment-versioning. Is there a way to make the link more prominent or make the title clickable so it's more obvious to click on something? cc @yalozhkin

Comment (dberenbaum) [reply src/components/Home/WhatsNewModal/index.tsx]: Also, let's slightly update the text.

Title: ML Experiment Versioning

Paragraph: Versioning experiments combines the benefits of version control and experiment tracking.

Comment (julieg18) [reply src/components/Home/WhatsNewModal/index.tsx]: > Okay, we've decided on https://dvc.org/blog/ml-experiment-versioning. Is there a way to make the link more prominent or make the title clickable so it's more obvious to click on something? cc @yalozhkin

Good idea! I'll turn the title into a link.

File: src/components/Home/WhatsNewModal/styles.module.css
Comment (jendefig) [reply src/components/Home/WhatsNewModal/styles.module.css]: Ok.  No worries then.  Today when Dave shared in our meeting, it looked good.  👍 May be my settings.

File: src/components/Home/WhatsNewModal/styles.module.css
Comment (jendefig) [reply src/components/Home/WhatsNewModal/styles.module.css:129]: Today at the meeting we discussed the contents of the modal:
1. This will be the link: https://dvc.org/blog/ml-experiment-versioning
2. Change Header to "ML Experiment Versioning"
3. Words to: "Versioning machine learning experiments combines the benefits of version control and experiment tracking. Read more."
4. Add pic from blog post into modal: Title, then pic, then text 

(cc: @dberenbaum for further confirmation)

File: src/components/Home/LandingHero/index.tsx
Comment (julieg18) [inline src/components/Home/LandingHero/index.tsx:116]: We need to update this to the new video once we have the id.

File: src/components/Home/WhatsNewModal/index.tsx
Comment (julieg18) [inline src/components/Home/WhatsNewModal/index.tsx:92]: I used a old blog image for now. We need to update this to the new blog image once we have it.

Comment (jendefig) [reply src/components/Home/WhatsNewModal/index.tsx:92]: It looks good!  Will the image be clickable too(link to the blog post?  And I think the text font could be a bit bigger if possible.  Maybe we should bold **Read more** too.

Comment (julieg18) [reply src/components/Home/WhatsNewModal/index.tsx]: > It looks good! Will the image be clickable too(link to the blog post? And I think the text font could be a bit bigger if possible. Maybe we should bold Read more too.

I turned the image into a link, increased the text size, and bolded the link!

**Comment by julieg18** [comment]
Landing page now has proper popup designs! Once we have a video id and blog image (and if we don't find anything wrong with the current code), we should be good to merge!

**Comment by jendefig** [comment]
@julieg18 Here is the link to the video.  It is currently not published.  It will publish at 3am PST tomorrow morning. 

https://youtu.be/z0s42TxH9oM

**Comment by jendefig** [comment]
@julieg18 Just noticed that in the modal it has the Community Gems image.  Is that just a place holder or that's because it's feeding from the current cover image from the blog and when the Dave's blog post goes live it will fill with the cover image?  Or do you need the image too? Clarifying just in case!

**Comment by julieg18** [comment]
> @julieg18 Just noticed that in the modal it has the Community Gems image. Is that just a place holder or that's because it's feeding from the current cover image from the blog and when the Dave's blog post goes live it will fill with the cover image? Or do you need the image too? Clarifying just in case!

It's a placeholder. I was planning to update the image as soon as Dave's post was merged to master

**Comment by jendefig** [comment]
> > @julieg18 Just noticed that in the modal it has the Community Gems image. Is that just a place holder or that's because it's feeding from the current cover image from the blog and when the Dave's blog post goes live it will fill with the cover image? Or do you need the image too? Clarifying just in case!
> 
> It's a placeholder. I was planning to update the image as soon as Dave's post was merged to master

Ok.  Cool.  I believe @dberenbaum will merge btwn 6am and 7 am ET tomorrow morning.
Thank you!!!

File: .stylelintrc
Comment (julieg18) [inline .stylelintrc:8]: I'm not sure why but I was forced to run prettier on several files when I merged to master 🤔

**Comment by jendefig** [comment]
When will this be merged @julieg18?

**Comment by julieg18** [comment]
> When will this be merged @julieg18?

@jendefig  We can technically merge it but I had an issue with our package.json and the commit check is failing. I'm pushing a possible fix now. I also need an approval to merge this :)

**Review by jendefig — APPROVED** [review]

File: .stylelintrc
Comment (rogermparent) [reply .stylelintrc:8]: The Gatsby 4 update also updated linters alongside it, and apparently those linters either weren't as stringent or weren't running at all before the update. That said, I don't know why changes that weren't originally in this PR are getting added by Prettier, they should have been taken care of by the retroactive fixes I did in that PR.

**Pull request merged**
